tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -244,6 +244,23 @@ spv::Op translateAtomicHlslOpcodeToSpirvOpcode(hlsl::IntrinsicOp opcode) {
   return Op::Max;
 }
 
+/// Returns true if the given function parameter can act as shader stage
+/// input parameter.
+inline bool canActAsInParmVar(const ParmVarDecl *param) {
```

Comment:
Hmm, actually I thought about that name before. It seems misleading to me since we have in/out/inout and a few others. `IsInParmVar` seems strictly testing `in`.

Reply:
SGTM